format/riff/dolby.go

```diff
@@ -0,0 +1,294 @@
+package riff
+
+// Dolby Metadata, e.g. Atmos, AC3, Dolby Digital [Plus]
+// https://tech.ebu.ch/files/live/sites/tech/files/shared/tech/tech3285s6.pdf
+// https://github.com/DolbyLaboratories/dbmd-atmos-parser
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/wader/fq/pkg/decode"
+	"github.com/wader/fq/pkg/scalar"
+)
+
+func dbmdDecode(d *decode.D, size int64) any {
```

Comment:
Oh hmm maybe you want to register this and adm as proper own formats? and then use them as subformat in the wav decoder? then they could be reused in other containers (is it used in mp4?) and be used with `-d` if that makes sense?